molecule/default/INSTALL.rst

```diff
@@ -0,0 +1,22 @@
+*******
```

Comment:
This file is not required for the test itself, can be removed I guess?

Reply:
Yes, you're right. I just copied over the default without checking this file. I'll delete it.